### PR TITLE
Add Traditional Chinese Language File.

### PR DESCRIPTION
**Pull request by jasoncheng7115**
Support Traditional Chinese.

### CODE REVIEW

File: src/lang/zh_TW.lang.php [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+		'email_smtp_host' => 'SMTP主機',
+		'email_smtp_port' => 'SMTP連接埠口',
+		'email_smtp_security' => 'SMTP安全性',
+		'email_smtp_security_none' => '無',
+		'email_smtp_username' => 'SMTP使用者名稱',
+		'email_smtp_password' => 'SMTP使用者密碼',
+		'email_smtp_noauth' => '留空白表示不做帳號驗證',
+		'sms_status' => '啟用簡訊通知',
+		'sms_gateway' => '簡訊服務提供者',
+		'sms_gateway_mosms' => 'Mosms',
```
Comment (TimZ99) [inline src/lang/zh_TW.lang.php]: The SMS gateways aren't in the translation files anymore.

**Review by TimZ99 — CHANGES_REQUESTED** [review]
You have 8 removed translations and 37 translations that are missing.
The translation files are based on the most recent English translation (en_US).

**Comment by TimZ99** [comment]
Thank you very much for the PR!
You can use [the translation editor](https://github.com/timz99/phpservermon-language-editor) to view the missing translations.

**Comment by jasoncheng7115** [comment]
Thanks! I have finished the Translation by using the Translation Editor and will re-submit the PR.

**Comment by TimZ99** [comment]
Thanks for the update, there are quite a lot of changes!

Just a few points:
The words: Port, Host, Error and IMPORTANT are not translated.
Same goes for alert_proxy and telegram_bot_username_found.
pushover_error_nokey (key found), telegram_description (documentation) and alert_proxy_url contain some English.
I will stop after this one, I promise 😅: the notifications (online messages) are missing ` it was down for %LAST_OFFLINE_DURATION%:`.

File: src/lang/zh_TW.lang.php [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+ * PHP Server Monitor is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PHP Server Monitor.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package     phpservermon
+ * @author      manhere <https://github.com/manhere>
```
Comment (TimZ99) [inline src/lang/zh_TW.lang.php]: You can put your own name here 😄

**Review by TimZ99 — CHANGES_REQUESTED** [review]

File: src/lang/zh_TW.lang.php (hunk 2 above)
Comment (jasoncheng7115) [reply src/lang/zh_TW.lang.php]: 😄 thanks, I've put it on.

**Comment by jasoncheng7115** [comment]
I will revise the missing part in your next release. ✌️✌️

**Comment by TimZ99** [comment]
Doesn't seem to get an update, so closing the pr. Feel free to reopen.

**Comment by TimZ99** [comment]
Thanks for your work. I merged this part and hopefully new additions will come.

**Pull request merged**